src/storage/benches/bench_compactor.rs

```diff
@@ -175,7 +175,7 @@ fn bench_table_scan(c: &mut Criterion) {
 
     c.bench_function("bench_table_iterator", |b| {
         let info1 = info.clone();
-        b.to_async(FuturesExecutor)
+        b.to_async(&runtime)
```

Comment:
I guess this is the main source of the bug?

Reply:
maybe